### PR TITLE
SERVER-25750: added VPAT (508) documentation

### PR DESCRIPTION
**Pull request by pdinc-oss**
- see: https://www.itic.org/policy/accessibility/
- based on data from https://www.mongodb.com/accessibility/vpat/mongodb24

### CODE REVIEW

**Comment by pdinc-oss** [comment]
jira ticket https://jira.mongodb.org/browse/SERVER-25750

File: docs/vpat.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+## Section 1194.41 Information, Documentation and Support – Detail
+ 
+|Criteria|Supporting Features|Remarks and explanations|
+|---|---|---| 
+|(a) Product support documentation provided to end-users shall be made available in alternate formats upon request, at no additional charge| Support documentation for this product is available in accessible electronic format or print format.| MongoDB documentation is available online: http://docs.mongodb.org/manual/. Alternately, users may, from the same link, access the documentation as single page HTML, epub or PDF format. There is no additional charge for these alternate formats.|
+|(b) End-users shall have access to a description of the accessibility and compatibility features of products in alternate formats or alternate methods upon request, at no additional charge.|Description is available in accessible electronic format online. |Information regarding accessibility and compatibility features are available online: www.10gen.com/legal/accessibility/VPAT Links to alternative formats may also be contained online, as applicable. There is no additional charge for these alternate formats.|
```
Comment (IanWhalen) [inline docs/vpat.md]: This doesn't seem like the right URL, but I'm not sure what the correct URL here should be.  A self-referential link to this document?

File: docs/vpat.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+|(c\) Support services for products shall accommodate the communication needs of end-users with disabilities.|10gen's support services provides two basic customer interfaces: (1) web support (https://jira.mongodb.org/browse/CS) and (2) phone support (646-201-9247). | 10gen customers primarily use Jira for communications with support. JIRA delivers content via a web interface that is accessible to existing screen readers. JIRA has been approved by a large range of US government and commercial organizations including the US Department of Energy, Department of the Interior, Department of Transport, Environmental Protection Agency, Navy and Supreme Court. JIRA Section 508 Accessibility Compliance is available here: https://confluence.atlassian.com/display/JIRACOM/JIRA+Section+508+Accessibility+Compliance |
```
Comment (IanWhalen) [inline docs/vpat.md]: 1) References to 10gen should be replaced with references to MongoDB, Inc.
2) We now offer web support via SalesForce Service Cloud at support.mongodb.com - you can find their VPAT at http://www.sfdcstatic.com/assets/pdf/misc/VPAT_servicecloud_summer2013.pdf.  We should also remove the reference to phone support as our updated phone contact info will be kept at support.mongodb.com too.

**Comment by pdinc-oss** [comment]
updated. regarding the circular reference, it is how it was done before. The assumption is that it is a "place" where more information may be added in the future.

**Comment by IanWhalen** [comment]
Committed as a00ac5695ffa65b0f4fe3089eb1943ff51b07b58 and 85da90453648d5d1d47a3f4d5aa45b58b526c4d4.  Thanks a lot for the contribution @pdinc-oss !

**Pull request closed without merging**